### ISSUE TITLE
Implement the "Search" page with all websites displayed

### ISSUE THREAD

**Issue by maxime-carabina** [massalabs/station#747]
![image](https://github.com/massalabs/thyra/assets/72019005/7ceedd9f-ad85-44de-ad1d-8bb9c1369cc2)

**Comment by SlnPons** [comment]
@Thykof @maxime-carabina When testing this page, i've noticed a couple of things that I would like to solve: 

1. The websites are NOT accessible. I can't click on them.
The expected behaviour is: 
As a user, when I click on the website block, I see a new tab open to the website I clicked on. 

2. The page takes a long time to display the websites. Is it expected ? (around 30sec I would say, web standard is more 1sec) 
Do you think we can reduce this time somehow? 
--> if not, we'll see for a UI solution later ;) - don't waste time on this now. 

3. Still no favicon and no news from you guys about what's possible or not.

Let me know if I can help you to unblock these points :)

**Comment by SlnPons** [comment]
@maxime-carabina I couldn't find a specific ticket for the side panel, so I'm sharing the output of my QA review here with you: 

Minor things to fix (it's all nice to have so it's not straighforward, let's move to sth else because I've stuff to correct elsewhere that are more important ;) ) 

* When opening the slider, the text field "name" looks selected by default 
Expected behaviour: text field is not selected (ie: not green) until user click on it. Default state = no green around.


![Image](https://github.com/massalabs/station/assets/109611779/9f3fb1a5-492c-42b4-8fcb-eda8d4a6127e)

* That's it. The rest is not perfect but ok for me :)

Well done !!

**Comment by maxime-carabina** [comment]
> @maxime-carabina I couldn't find a specific ticket for the side panel, so I'm sharing the output of my QA review here with you:
> 
> Minor things to fix (it's all nice to have so it's not straighforward, let's move to sth else because I've stuff to correct elsewhere that are more important ;) )
> 
> * When opening the slider, the text field "name" looks selected by default
>   Expected behaviour: text field is not selected (ie: not green) until user click on it. Default state = no green around.
> 
> ![Image](https://user-images.githubusercontent.com/109611779/247539218-9f3fb1a5-492c-42b4-8fcb-eda8d4a6127e.png)
> 
> * That's it. The rest is not perfect but ok for me :)
> 
> Well done !!

@SlnPons  This problem will be corrected with the addition of the upload (#816).

**Comment by SlnPons** [comment]
Output of the QA about the upload website function: 

* the theme is broken on main following the merge of this function


![Image](https://github.com/massalabs/station/assets/109611779/d0758d91-cbe0-4699-94eb-683d00a23718)

* the account selection dropdown should only appear on the search page (ideally please, if not possible then let's catch up to see what we can do but it's really weird to have this on the home page....) 

* the account selection dropdown behaviour when no wallet exist is KO

![Image](https://github.com/massalabs/station/assets/109611779/a662f5cc-a45b-430a-b85c-56329b770bae)

expected: 


![Image](https://github.com/massalabs/station/assets/109611779/f9f3034e-c597-4a4c-bdb5-2a50fc6bf8c5)


* I can't see the validation rules in the front-end for the description (280 characters max)
* I can't see the validation rules (I think both BE & FE) for the DNS name already existing. I've got an error in the console but nothing shown in the FE


![Image](https://github.com/massalabs/station/assets/109611779/f74f68e3-9069-4346-9711-3eb412976c57)

Expected: I can't have 2 websites with the same name. Even if the description is different.

* When trying to upload a website - I can't see the spinner however, see error reported below


* I can't upload a website, i've an error 500: 


![Image](https://github.com/massalabs/station/assets/109611779/ed8b4744-94bb-46e1-9073-a671887b36b2)

**Comment by mazmassa** [comment]
> Output of the QA about the upload website function:
> 
> * the theme is broken on main following the merge of this function
> 
> ![Image](https://user-images.githubusercontent.com/109611779/248010346-d0758d91-cbe0-4699-94eb-683d00a23718.png)

The first one concerning the images I already fixed and addressed here: https://github.com/massalabs/station/pull/853

**Comment by SlnPons** [comment]
see https://github.com/massalabs/station/pull/783 for the BE validation rules if that's help :)

**Comment by Thykof** [comment]
PR to fix QA review:

- https://github.com/massalabs/station/pull/861
- https://github.com/massalabs/station/pull/862
- https://github.com/massalabs/station/pull/863
